geoh5py/shared/concatenation/drillholes_group_table.py

```diff
@@ -205,6 +206,16 @@ def add_values_to_property_group(
                 f"({self.parent.data[self.association[0]].shape})."
             )
 
+        if isinstance(value_map, dict):
+            data_type = DataType(
+                self.parent.workspace, "REFERENCED", name=name, value_map=value_map
+            )
+        else:
+            data_type_ = DataType.validate_data_type(
+                self.parent.workspace, {"values": values}
+            )["primitive_type"]
+            data_type = DataType(self.parent.workspace, data_type_, name=name)
```

Comment:
pass DataType

Reply:
I agree, but petro-lingo is now using value_amp.
Can we do another issue after this one is resolved to correct it both in geoh5py and petro-lingo to prevent bugs?